Spigot-Server-Patches/0551-Optimize-Villagers.patch

```diff
@@ -38,30 +38,42 @@ the code rebuilding data that is static every single invocation for every POI ty
 So we cache that and only rebuild it if professions change, which should be never unless
 a plugin manipulates and adds custom professions, which it will handle by rebuilding.
 
+Ported to 1.16.1 by MrIvanPlays
+Co-authored-by: MrIvanPlays <ivan@mrivanplays.com>
```

Comment:
It's cool to want attribution, but you didn't co-author this patch.

Reply:
I want to see this commented from a papermc member otherwise it would stay.

Reply:
Multiple people have ported patches over to new major MC versions countless times. They've never placed their name inside the patch for credit.

Your "port" is no different in that regard, git commit history is here for that.

Reply:
Do not slap your name on things you haven't done. This is also the reason why you were banned from the IRC or Paper's Discord Guild.

Reply:
I think his confusion comes from @Proximyst saying "If it was based on someone else's but you didn't write the base, it's still their patch. You'd write you co-authored it in that case." In response to Ivan saying "The Pathfinder changes don't exactly match aikar's changes in the previous patch."

Reply:
pretty sure the reason behind why you don't want it there is because i'm banned from paper discord and you don't want to see me as a contributor. as i said i want to see this commented from papermc member (prefferably, the author of the patch - aikar) and then ill remove it.

> This is also the reason why you were banned from the IRC or Paper's Discord Guild.

It wasn't for that, no one minded I stole 1 patch of your microcord which later I removed. Now I did change and im trying to show it off by starting to contribute things and being more active in the overall minecraft discords, but the paper discord - no one wants to talk with me in other discords and i can't show it off to them.

> Multiple people have ported patches over to new major MC versions countless times. They've never placed their name inside the patch for credit.

valid point, but still want to see it commented from a papermc member.

Reply:
Why are you still arguing about this when it's clear that you're wrong? It doesn't make sense to wait for Aikar to tell you the exact same thing 5 people have already told you. I don't think anyone cares whether you are banned on the Discord or not, contributing is great, and everyone loves you for that. Just don't insist that you put your name on something that isn't your work.

Reply:
> you don't want to see me as a contributor

Uh.. Even just with 1 commit you will end up in the list of contributors.. https://github.com/PaperMC/Paper/graphs/contributors

Best to handle things the right way, you're just making an clown out of yourself how you're doing it now.

Reply:
> and everyone loves you for that

nice joke. i still think you're hating me no matter what. even when mini was porting paper to 1.16.1 and started a test server some1 just banned me for no reason. you all fucking hate me, you can stop trying to "show-off" that you actually care about the patch. you only care me to not be a contributor and to NOT participate in anything paper-related.

Reply:
Ivan, can you stop making a fool of yourself for 3 minutes?

You didn't author nor coauthor the patch, Aikar did. You updated it. They are asking you to remove your coauthor attribution because you are not the coauthor of the patch.

This is not in any way different from you choosing any book, updating it, and then slapping your name next to the author. It wouldn't be true and neither is this.

You're free to contribute. If you weren't, you'd be banned from even making issues/PRs so seriously, stop.

You claim all day and night that you've changed, but you literally have not. You love to spout off about how we're targeting you but you are doing specific actions repeatedly that do not align with Paper's goals. If you don't stop, then yeah you're going to get banned and nobody is going to like your attitude.

But that doesn't preclude you from still taking part of the community if you change your ways.

My 2c. Whatever. I doubt anyone cares so much about this patch that they are willing to do drama over it. 

TLDR remove the coauthor status or close the PR, but stop being so annoying.

Reply:
Brb, adding myself as co-author to the ~200 patches I updated to 1.16

Reply:
There is nothing significant changed here to warrant a co author. So please remove.

Authorship is more about ideas and implementation design, not the syntax of a line.

Reply:
It was already removed with https://github.com/PaperMC/Paper/pull/4101/commits/ce978bbff0d937df7a966474779a4ea4c52bbabb